gulp/config.js

```diff
@@ -50,14 +50,15 @@ module.exports = function () {
   };
 
   var appFiles = {
-    scripts:  paths.scripts.src + '**/*.js',
-    styles:   paths.styles.src + '**/*.scss',
-    content:  paths.content.src + '**/*.yml',
-    pages:    paths.pages.src + '**/*.jade',
-    images:   paths.images.src + '**/*',
-    logos:    paths.logos.src + '**/*',
-    favicons: paths.favicons.src + '**/*',
-    fonts:    paths.fonts.src + '**/*'
+    scripts:   paths.scripts.src + '**/*.js',
+    styles:    paths.styles.src + '**/*.scss',
+    content:   paths.content.src + '**/*.yml',
+    pages:     paths.pages.src + '**/*.jade',
+    pagesHtml: paths.pages.dest + '**/*.html',
```

Comment:
We should come up with a better name.
Perhaps even remove this as it's the only non source files.
Why not just use paths.pages.dest + '**/*.html' in the task?